index.md

```diff
@@ -5,19 +5,43 @@
 # Look in _data/ to modify actual data.
 layout: home
 ---
-## Welcome to the Hands-On UNIX SysAdmin DeCal, Fall 2017 edition!
-This course will cover the basics of setting up and administering a production-quality Linux-based server environment in the manner of the EECS instructional machines (hivexx.cs.berkeley.edu) and those of the Open Computing Facility (tsunami.ocf.b.e). By the end of this course, we expect you to be completely comfortable with:
+## Welcome to the Hands-On UNIX SysAdmin DeCal, Spring 2018 Edition!
+This course will cover the basics of setting up and administering a
+production-quality Linux-based server environment in the manner of the EECS
+instructional machines (hivexx.cs.berkeley.edu) and those of the Open Computing
+Facility (tsunami.ocf.b.e). By the end of this course, we expect you to:
 
-* interfacing with Linux
-* be broadly familiar with various fundamental things like services and daemons
+* be completely comfortable using GNU/Linux
+* have a general understanding of the various parts of the OS,
+  e.g. init, filesystems, services, daemons, etc.
 * understand basic networking
-* have a good sense about
-* maintaining system security
+* have a good sense about maintaining system security
 * understand system administration essentials
-* get a practical taste for what's new in the field
+* get a practical taste of what sysadmins do in industry.
 
-Lectures are scheduled for **8-9pm Tuesdays and Thursdays** in the OCF Lab, located at **171 MLK** (MLK basement down the hall from the student store). **Attendance is mandatory with two allowed unexcused abscences** (other conflicts such as exams can be excused).
+Lectures are scheduled for **Tuesdays and Thursdays 8-9pm** the OCF Lab,
+located at **171 MLK** (MLK basement down the hall from the student store).
+**Attendance is mandatory, with two allowed unexcused absences.**
+(other conflicts such as exams are excused).
```

Comment:
I'd say something like `some other conflicts, such as exams, are excused`, since some conflicts would be excusable, and some would not.